performances/irobot-benchmark/src/irobot-benchmark.cpp

```diff
@@ -26,6 +26,68 @@
 
 using namespace std::chrono_literals;
 
+template <typename NodeT>
+void init_ros_system(int argc, char** argv,
```

Comment:
The name of this function is wrong.
It's not initializing the ROS system, it's also doing the whole experiment and saving statistics.